src/dom/DomEvent.js

```diff
@@ -81,7 +81,7 @@ function addOne(obj, type, fn, context) {
 
 	var originalHandler = handler;
 
-	if (Browser.pointer && type.indexOf('touch') === 0) {
+	if ((Browser.pointer || Browser.edge) && type.indexOf('touch') === 0) {

```

Comment:
I would say that `if (Browser.pointer && type.indexOf('touch') === 0) {`
is correct and should not be changed. 

The line i think we need to change is the line 92 of [https://github.com/Leaflet/Leaflet/blob/master/src/core/Browser.js](https://github.com/Leaflet/Leaflet/blob/master/src/core/Browser.js).
The line `export var msPointer = !window.PointerEvent && window.MSPointerEvent;` should be changed to
`export var msPointer = window.PointerEvent || window.MSPointerEvent;` The reason for this is that both Edge and IE11 have `window.PointerEvent` but only IE11 `window.MSPointerEvent`

You can test it out here: [https://github.com/ggcaponetto/Leaflet/tree/pull-6897-zoom-pan-win10-chrome](https://github.com/ggcaponetto/Leaflet/tree/pull-6897-zoom-pan-win10-chrome).